tic-tac/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+---
+linters:
+   rubocop:
+       display_cop_names: true
+       fixer: true
```

Comment:
### `REQUIRED`

The `.rubocop.yml` is having the wrong content. Use this instead. You can find the file [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)

```yml
AllCops:
  Exclude:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
  DisplayCopNames: true

Metrics/LineLength:
  Max: 120
Metrics/MethodLength:
  Include: ["app/controllers/*"]
  Max: 20
Metrics/AbcSize:
  Include: ["app/controllers/*"]
  Max: 30
Metrics/ClassLength:
  Max: 150
Metrics/BlockLength:
  ExcludedMethods: ['describe']

Style/Documentation:
  Enabled: false
Style/ClassAndModuleChildren:
  Enabled: false

Layout/AlignHash:
  EnforcedColonStyle: key
Layout/ExtraSpacing:
  AllowForAlignment: false
Layout/MultilineMethodCallIndentation:
  Enabled: true
  EnforcedStyle: indented
```